src/utils/predictoor.js

```diff
@@ -0,0 +1,26 @@
+export const getPredictoorAccuracy = async (userAddress) => {
+    let res;
+    try {
+      res = await fetch(`${import.meta.env.VITE_BACKEND_API}/predictoor/data`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          query: {
+                round: {
+                  $gt: -1,
+                }
+            },
+          fields: [
+            "accuracy",
+          ]
+        }),
+      });
+    } catch (error) {
+      console.log(error);
+      return 0;
+    }
+    let data = await res.json();
+    return data.length > 0 ? data.legth : 0
```

Comment:
Pretty sure this query & results won't work.
1. You can't get accuracy from `/predictoor/data`. You should use predictoorSummary instead.
2. You're returning the length of the result, not the accuracy of the user. If you use the method above (1) you're also only going to report `per-round-accuracy` (i.e. accuracy of the last round), rather than `accuracy-average-lifetime`.

![image](https://github.com/oceanprotocol/df-web/assets/69865342/52f787c0-5963-4cd9-9679-b11a540f22bf)

I would recommend we report a simple "prediction record" as the metric.

`${sum(correct_prediction_count)} / ${sum(total_predictions)}`

Reply:
I get the data from `/predictoor/summary`, and it displays as you mentioned the last round accuracy.
Maybe we can discuss a little bit if the metric should be lifetime average or per round. I personally think per round is more valuable because you can then compare it to previous rounds, or with other users, as for a lifetime metric is hard to tell how you performed during a round.
I would also like to have the number of correct predictions / total predictions because it's not really valuable for me if I have 70% accuracy but only 50 predictions to compare with someone else who has 65% but 2000 predictions only based on accuracy, the number of predictions also matter.